### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by oleksij-omelchenko**
[DEMO LINK](https://oleksij-omelchenko.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://oleksij-omelchenko.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,12 +3,60 @@
...
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"
+        class="header__icon">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:21]: Check the code style everywhere. No need to wrap attributes if there are 2 or less
```suggestion
      <a href="#home" class="header__icon">
```

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,60 @@
...
+        <img src="./images/logo.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:26]: Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </a>
      
      <nav class="nav">
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
+ul {
```
Comment (Anastasiia-Svintsova) [inline src/style.css:5]: use class names instead

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,85 @@
...
+  --color: #00acdc;
+}
+
+.header {
+  min-width: 1024px;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:27]: Seems that background is white by default

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,85 @@
...
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__icon {
+  width: 40px;
+  height: 40px;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:33]: You can create a variable for the logo size as well

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,85 @@
...
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:39]: Height is 100% by default. You can remove redundant styles. The display property seems redundant as well

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,85 @@
...
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  height: 100%;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:45]: The same about the height property

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,85 @@
...
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
+}
+
+.nav__item {
+  display: flex;
+
+}
```
Comment (Anastasiia-Svintsova) [inline src/style.css:61]: Seems redundant

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Good start! Check my comments, please

File: src/index.html [hunk 9]
```diff
@@ -3,12 +3,64 @@
...
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#home"
```
Comment (polosanya) [inline src/index.html:21]:
```suggestion
    <header class="header">
      <a href="#home"
```
Don't add empty line between parent and child elements

File: src/index.html [hunk 10]
```diff
@@ -3,12 +3,64 @@
...
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="hover">
+              Laptops & Computers
+            </a>
```
Comment (polosanya) [inline src/index.html:46]:
```suggestion
            <a 
              href="#Laptops & Computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
Fix all cases
image

**Review by polosanya — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes?

If you have any questions, please ask them in fe_chat and tag mentors 🙂

**Pull request closed without merging**